daemon/oci_linux.go

```diff
@@ -463,11 +463,9 @@ func setMounts(daemon *Daemon, s *specs.Spec, c *container.Container, mounts []c
 		}
 
 		if m.Source == "tmpfs" {
-			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode}
+			opt := []string{"noexec", "nosuid", "nodev", volume.DefaultPropagationMode, "size=65536k"}
```

Comment:
Thanks @cpuguy83 I just added a test for custom propagation mode. Since busybox does not have the `findmnt` I used `debian:jessie` so that 
```
 findmnt -o TARGET,PROPAGATION /tmp
```
could be specified.

I also take a look at the mount command:
http://man7.org/linux/man-pages/man8/mount.8.html
> The usual behavior is that the last option wins if there are conflicting ones.

So I think it should be ok to having duplicate options as long as the last one is the one we want.